core/utils.ts

```diff
@@ -6,34 +6,40 @@ import { Resolvable } from "df/core/common";
 import { IActionProto, Session } from "df/core/session";
 import { dataform } from "df/protos/ts";
 
-export const pathSeperator = (() => {
-  if (typeof process !== "undefined") {
-    return process.platform === "win32" ? "\\" : "/";
-  }
-  return "/";
-})();
+export class Path {
```

Comment:
using a `class` as a namespace is really Java-y.

just use normal TypeScript style, `import * as path from "...path.ts"`